src/index.html

```diff
@@ -11,7 +11,137 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form
+    action="Create HTML page with form. On form submit send form data to https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    >
+      <fieldset class="block">
+        <legend>Personal information</legend>
+        <div class="class">
+          <label for="surname">Surname:</label>
+          <input
+          type="text"
+          name="Surname"
+          id="surname"
+          minlength="5"
+          maxlength="50"
+          autocomplete="off"
+          >
+        </div>
+        <div class="class">
+          <label for="name">Name:</label>
+          <input
+          type="text"
+          name="Name"
+          id="name"
+          minlength="5"
+          maxlength="50"
+          autocomplete="off"
+           >
+        </div>
+        <div class="class">
+          <label for="age">How old are You?</label>
+          <input
+          type="number"
+          name="number"
+          id="age"
+          min="1"
+          max="100"
+          value="12"
+          >
+        </div>
+        <div class="class">
+          <label for="dateOfBirth">Full date of birth:</label>
+          <input
+          type="date"
+          name="dateOfBirth"
+          id="dateOfBirth"
+          >
+        </div>
+        <div>
+          <label for="agreement">I accept the term of the agreement:</label>
+          <input
+          type="checkbox"
+          name="Agree"
+          id="agreement"
+          required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="block">
+        <legend>Registration</legend>
+        <div class="class">
+          <label for="mail">E-mail:</label>
+          <input
+            type="email"
+            name="mail"
+            id="mail"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+        <div>
+          <label for="password">Password:</label>
+          <input
+          type="password"
+          name="password"
+          id="password"
+          required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="block">
+        <legend>An interesting fact about you!</legend>
+        <div class="class">
+          Do you love cats?
+          <input type="radio" name="Yes" id="yes">
```

Comment:
If you have more than 2 attributes, put each of them on a newline:
```suggestion
          <input
            type="radio"
            name="Yes"
            id="yes"
          >
```